src/main/java/controllers/ApplicationController.java

```diff
@@ -71,14 +71,23 @@ public Result dealspPost(Context context, spGame g2) {
         return Results.json().render(g2);
     }
 
-    public Result removeCard(Context context, @PathParam("column") int colNumber, Game g){
+    public Result removeamCard(Context context, @PathParam("column") int colNumber, amGame g){
         g.remove(colNumber);
         return  Results.json().render(g);
     }
 
-    public Result moveCard(Context context, @PathParam("columnFrom") int colFrom, @PathParam("columnTo") int colTo, Game g){
+    public Result removespCard(Context context, @PathParam("column") int colNumber, spGame g){
```

Comment:
The previous function had a parameter of game object, but the design pattern we chose had game be an abstract class. I had to create two different functions for each game, one that accepted the american game and one that accepted the spanish game